firmware/ventilator-controller-stm32/Core/Test/Pufferfish/Protocols/messages.txt

```diff
@@ -0,0 +1,181 @@
+/* Write function
+Scenario: Message: The write method correctly generates an encoded body from the protobuf message descriptor payload
```

Comment:
All tests in this scenario need to describe what the value of the `type` and `payload` members should be after `write` is called.